min/lib/Minify.php

```diff
@@ -517,7 +518,7 @@ protected static function _combineMinify()
                 // minify previous sources with last settings
                 $imploded = implode($implodeSeparator, $groupToProcessTogether);
                 $groupToProcessTogether = array();
-                if ($lastMinifier) {
+                if ($lastMinifier && !$no_minify) {
```

Comment:
This isn't a clean way to do this, and will prevent URIs from being rewritten in CSS, which is necessary even when not minifying. I'd rather set up the source objects so they don't do compression.